redact/redact_requests.py

```diff
@@ -268,8 +268,9 @@ def get_labels(
         )
 
         debug_uuid = uuid.uuid4()
+        # Added timeout due to CT-660: Large labels files lead to trigger the 5s default
         response = self._retry_on_network_problem_with_backoff(
-            self._client.get, debug_uuid, url, headers=self._headers
+            self._client.get, debug_uuid, url, headers=self._headers, timeout=60.0
```

Comment:
Wouldn't it make more sense to add this as an input parameter for the function `get_labels` instead of hardcoding the value here. We can then adjust the value in the future instead of always creating a new release for the redact client. Also I think it would make sense to add the parameter also to the normal function for downloading a file since that could also happen there.

Reply:
Yes, I believe this is a first fix but I created CT-660 to investigate create a better approach towards timeouts and the label processing. Adding now a lot of parameters raises the question where we really need them to be configurable. And that we first need to answer with CT-660. Some possible answers: Function param, settings, env variable, auto-discover?